File: 백준/Bronze/5613. 계산기 프로그램/계산기 프로그램.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        ArrayList<String> row = new ArrayList<String>();
        row.add("+");
        while (true) {
            String s = sc.next();
            if (s.equals("=")) {
                break;
            }
            row.add(s);
        }
        int cnt = 0;
        for (int i = 0; i < row.size(); i += 2) {
            if (row.get(i).equals("+")) {
                cnt += Integer.parseInt(row.get(i + 1));
            } else if (row.get(i).equals("-")) {
                cnt -= Integer.parseInt(row.get(i + 1));
            } else if (row.get(i).equals("*")) {
                cnt *= Integer.parseInt(row.get(i + 1));
            } else {
                cnt /= Integer.parseInt(row.get(i + 1));
            }
        }
        System.out.println(cnt);
    }
}

File: 백준/Bronze/5613. 계산기 프로그램/계산기 프로그램.py
row = ['+']
while True:
    s = input()
    if s == '=':
        break
    row.append(s)

cnt = 0
for i in range(0, len(row), 2):
    if row[i] == "+":
        cnt += int(row[i + 1])
    elif row[i] == "-":
        cnt -= int(row[i + 1])
    elif row[i] == "*":
        cnt *= int(row[i + 1])
    else:
        cnt //= int(row[i + 1])
        
print(cnt)